presto-hive/src/main/java/com/facebook/presto/twitter/hive/MetastoreStaticClusterModule.java

```diff
@@ -11,8 +11,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.facebook.presto.hive;
+package com.facebook.presto.twitter.hive;
```

Comment:
MetastoreStaticClusterModule was only made so that the module can be loaded dynamically. https://github.com/twitter-forks/presto/pull/5
This might be something that can live in FB namespace.